modules/api/src/main/java/org/apache/ignite/app/Ignite.java

```diff
@@ -15,7 +15,18 @@
  * limitations under the License.
  */
 
+package org.apache.ignite.app;
+
+import org.apache.ignite.table.manager.TableManager;
+
 /**
- * Contains internal tests or test related classes and interfaces.
+ * Ignite node interface. Main entry-point for all Ignite APIs.
  */
-package org.apache.ignite.internal.testframework;
+public interface Ignite extends AutoCloseable {
```

Comment:
It needs to add also a configuration manager here.

Reply:
I believe it'll be added during table creation implementation.